bin/opx-config-lag

```diff
@@ -116,14 +109,6 @@ class cli(object):
         self.set_member_ports(sorted(member_ports),'set')
 
 
-        # Assign unblocked ports
```

Comment:
What happened to the unblocked ports? If you're getting rid of this section, then shouldn't you also remove it from the list of args?

Reply:
@jeff-yin 
- For the create operation, any member port is by-default unblocked. So we need not explicitly add them to the unblocked-port list. 
- unblockedports argument is still needed. Without that we cant specify the ports which are 'not' blocked.